default/scripting/buildings/GATEWAY_VOID.focs.txt

```diff
@@ -34,7 +34,7 @@ BuildingType
                 InSystem id = Source.SystemID
             ]
             activation = Turn low = Source.CreationTurn + 1
-            priority = [[EARLY_POPULATION_OVERRIDE_PRIORITY]]  // implies overriding target pop effects
+            priority = [[EARLY_POPULATION_OVERRIDE_PRIORITY]]  // also implies overriding target pop effects
```

Comment:
still not clear to me what this means. is it that this priority will be after both target and current population, and thus override both? I think the use of "implies" is making it unclear.